functions.js

```diff
@@ -1 +1,112 @@
-// YOUR CODE HERE
+// ### maxOrMin(num1, num2, max)
+// Define a function maxOrMin that takes three parameters: two numbers and a
+// boolean. Have it return the larger of the two numbers if the boolean is true,
+// otherwise have it return the lesser of the numbers.
+
+function maxOrMin(a, b, boolean) {
+  // if boolean is true & a is greater than b, return a
+  if ( (boolean) && (a > b) ) {
+    return a;
+  // if boolean is true & b is greater than a, return b
+  } if ( (boolean) && (b > a) ) {
+    return b;
+  // if boolean is not true & a is less than b, return a
+  } else if ( (!boolean) && (a < b) ) {
+    return a;
+  // if boolean is not true & b is less than a, return b
+  } else if ( (!boolean) && (b < a) ) {
+    return b;
+  }
+}
+
+maxOrMin(1, 2, "");
+
```

Comment:
This works. You are passing an empty string, which is falsy, as the 3rd argument and the results are correct, but we would probably expect true or false in that 3rd slot.  Props if you were testing truthy and falsy values